node/src/stream_handler_pool.rs

```diff
@@ -326,6 +327,7 @@ impl StreamHandlerPool {
                     .expect("Neighborhood is Dead")
             }
             Endpoint::Socket(socket_addr) => {
+                // The socket_addr can either be for the Neighbor or the browser
```

Comment:
This is something that Syther and I concluded when we went through a deep dive of the codebase and looked into the every usage and decided that it's appropriate to put a comment over here.

I believe it'll definitely help any developer who would look into this piece of code because they want to know where exactly it's coming from. Also, if the usage changes, the developer will feel the necessity to change the comment appropriately to save them from any confusion.